server/metagraphs.go

```diff
@@ -161,3 +165,32 @@ func (server *GripServer) addFullGraph(ctx context.Context, graphName string, sc
 	}
 	return nil
 }
+
+func (server *GripServer) LoadSchemas(project_id string, sch *gripql.Graph, out *graph.GraphSchema) (*graph.GraphSchema, error) {
+	schcompiler := jsonschema.NewCompiler()
+	schcompiler.ExtractAnnotations = true
+	schcompiler.RegisterExtension(compile.GraphExtensionTag, compile.GraphExtMeta, compile.GraphExtCompiler{})
```

Comment:
Our graph extension is hard coded into the loader here. An extension argument could get abstracted out as an argument to the load schema protobuf method.